examples/cognite_core/_api/_core.py

```diff
@@ -205,14 +206,23 @@ def _iterate(
         limit: int | None,
         retrieve_connections: Literal["skip", "identifier", "full"],
         sort: list[InstanceSort] | None = None,
+        cursors: dict[str, str | None] | None = None,
```

Comment:
what happens if the value is None?

Reply:
It does not have a cursors for that part of the query